### PR TITLE
client: add iOS icon

### LINKED ISSUE #2939: Missing system icon for iOS ‘add to home screen’ functionality
image

### PR DESCRIPTION
**Pull request by kontrollanten**
## Description
iOS wants it's own markup to display custom icons on the home screen.

## Related issues
Closes #2939

## Has this been tested?
No, I couldn't find any way to test it other than create an own release and deploy it to some server. I hope this is okay anyway since the risks are low.

- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by test2a** [comment]
Screenshot_20210104-090726_1
i just checked peertube3.cpy.re, peervideo.ru on my android 8 running firefox and using the "install" feature. the icons have the orange background mixing with the middle triangle of the logo... can this be fixed?

**Comment by kontrollanten** [comment]
I'd say that configuring the background color is a separate issue. Strange though that the background color in the manifest (https://peertube3.cpy.re/manifest.webmanifest) is set to white.

File: client/src/index.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
     <link rel="manifest" href="/manifest.webmanifest?[manifestContentHash]">
 
     <link rel="icon" type="image/png" href="/client/assets/images/favicon.png?[faviconContentHash]" />
+    <link rel="apple-touch-icon" href="/client/assets/images/icons/144x144.png" sizes="144x144" />
```
Comment (Chocobozzz) [inline client/src/index.html:13]: These paths do not work

**Review by Chocobozzz — CHANGES_REQUESTED** [review]

**Comment by Chocobozzz** [comment]
Please use release/3.0.0 as target branch :)

**Comment by Chocobozzz** [comment]
Thanks @kontrollanten

**Pull request merged**